packages/core/src/node-utils/manifest.ts

```diff
@@ -1,10 +1,14 @@
 import Files from "./files";
 import { FileContent, SMConfigPath } from "./paths";
 import { Manifest } from "../models/Manifest";
+import { pipe } from "fp-ts/lib/function";
+import { fold } from "fp-ts/lib/Either";
+import { formatValidationErrors } from "io-ts-reporters";
+import * as t from "io-ts";
 
 export function createManifest(cwd: string, manifest: Manifest): void {
   const manifestPath = SMConfigPath(cwd);
-  Files.write(manifestPath, JSON.stringify(manifest, null, "\t"), {
+  Files.write(manifestPath, JSON.stringify(manifest, null, 2), {
```

Comment:
the Files write helper already stringify and format so it can be:
```
Files.write(manifestPath, manifest, { recursive: false })
```

Reply:
k